style.css

```diff
@@ -0,0 +1,105 @@
+* {
+    box-sizing: border-box;
+}
+a {
```

Comment:
saying it here but please change throughout this file.
please leave a line break between end of one css rule and the start of another